packages/core/src/context/MemoryStore.ts

```
@@ -0,0 +1,443 @@
+    this.memoriesCache.clear();
+  }
+
+  /**
+   * Get memories from a specific file
+   */
+  async getMemoriesFromFile(filename: string): Promise<Memory[]> {
+    const filePath = path.join(homedir(), GEMINI_CONFIG_DIR, filename);
+    
+    // Check cache first
+    if (this.memoriesCache.has(filePath)) {
+      return this.memoriesCache.get(filePath)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)